src/components/notifications/noti-buttons.css

```
@@ -0,0 +1,91 @@
+oui-noti-buttons>button#buttonUnread, oui-noti-buttons>#buttonRead, oui-noti-buttons>#buttonUnused {
+  transition: all 0.05s;
+  cursor: pointer;
+  outline: none;
+  -webkit-transition-timing-function: ease-in-out;
+  transition-timing-function: ease-in-out;
+}
+
+oui-noti-buttons>button#buttonUnread:hover, oui-noti-buttons>button#buttonRead:hover {
+  transform: scale(1.05);
+  box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.2);
+  background-color: rgb(237, 237, 237);
+  border: solid rgb(237, 237, 237);
+}
+
+oui-noti-buttons>button#buttonUnused:hover {
+  transform: scale(1.05);
+  box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.2);
+  background-color: rgb(237, 237, 237);
+  /* border: solid rgb(237, 237, 237); */
+}
+
+oui-noti-buttons>button#buttonUnread:active, oui-noti-buttons>button#buttonRead:active, oui-noti-buttons>button#buttonUnused:active {
```

Very long line. Breack multiple selectors into separate lines.

```suggestion
oui-noti-buttons>button#buttonUnread:active,
oui-noti-buttons>button#buttonRead:active,
oui-noti-buttons>button#buttonUnused:active {
```

---

Done :)